client/public/index.html

```diff
@@ -33,6 +33,12 @@
     <title>React App</title>
     <!-- Fonts to support Material Design -->
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
+
+    <style>
+      body {
+        background-color: #fafafa !important;
```

Comment:
`!important` is usually bad because it violates the cascading nature of CSS - a child may not be able to apply styling because a parent has an `!important` style.

I used it here as a quick and dirty method to see how we like the light gray background. Later on I'll do it in a more MUI friendly way so that dark theme works too.